CONTRIBUTING.md

```diff
@@ -11,10 +11,10 @@ The procedure for actually contributing is fairly simple:
   * If you're fixing a bug, reference the issue that the bug was first reported in somewhere in your pull request
 * I'll either approve your changes or let you know if there's any problems.
 
-# List of Contributors
+# List of Primary Rewrite Contributors
```

Comment:
Well... this is the third rewrite...

I'm not sure this section is still useful, though, or that it was ever useful? Like, people can click then "contributors" tab in the repo

Reply:
Yeah this seems like more work then need be, I'll just scrap it.